### PR TITLE
Add run status in action view page

### PR DESCRIPTION
**Pull request by yp05327**
Before:
image
image

After:
image
image

### CODE REVIEW

**Review by wolfogre — APPROVED** [review]

**Comment by lunny** [comment]
Looks like the status icon is not aligned with the title.

**Comment by yp05327** [comment]
> Looks like the status icon is not aligned with the title.

@lunny 
It looks good now.
image

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -2,7 +2,13 @@
   <div class="action-view-container">
     <div class="action-view-header">
       <div class="action-info-summary">
-        {{ run.title }}
+        <SvgIcon name="octicon-check-circle-fill" size="20" class="green" v-if="run.status === 'success'"/>
+        <SvgIcon name="octicon-clock" size="20" class="ui text yellow" v-else-if="run.status === 'waiting'"/>
+        <SvgIcon name="octicon-meter" size="20" class="ui text yellow" class-name="job-status-rotate" v-else-if="run.status === 'running'"/>
+        <SvgIcon name="octicon-x-circle-fill" size="20" class="red" v-else/>
```
Comment (delvh) [inline web_src/js/components/RepoActionView.vue:8]: What about separating a `RunStatus` component that takes as parameter the `status` (and perhaps others like `size` and `extraClasses` if we really need these differences)?
Then we automatically keep this one and the one below in sync…

Comment (yp05327) [reply web_src/js/components/RepoActionView.vue:8]: This is a good idea. How about merge this PR first, and I will try to do this in another PR.

PS: It seems that `SvgIcon` are not used in:
image
image
Are these codes removable?

Comment (yp05327) [reply web_src/js/components/RepoActionView.vue:8]: @delvh 
https://github.com/go-gitea/gitea/pull/23259
I created a new PR to do this. I can merge it to this PR, if it looks good.

**Review by lunny — APPROVED** [review]

File: web_src/js/components/RepoActionView.vue (hunk 1 above)
Comment (lunny) [reply web_src/js/components/RepoActionView.vue:8]: Both OK from me to keep that PR standalone or merge it into this one.

**Comment by yp05327** [comment]
Please remove `backport/manual` tag, as #24223 is merged.

**Comment by silverwind** [comment]
@yardenshoham what is the correct method to prevent re-addition of `backport/manual` like above? Set to `backport/done`?

**Comment by silverwind** [comment]
Seems like adding `backport/done` works :)

**Comment by yardenshoham** [comment]
Yes, if you manually create a backport you should add `backport/done`

**Pull request merged**